probatus/feature_elimination/feature_elimination.py

```diff
@@ -385,9 +387,7 @@ def _get_feature_shap_values_per_fold(
         y_train, y_val = y.iloc[train_index], y.iloc[val_index]
 
         if sample_weight is not None:
-            clf = clf.fit(
-                X_train, y_train, sample_weight=sample_weight.iloc[train_index]
-            )
+            clf = clf.fit(X_train, y_train, sample_weight=sample_weight.iloc[train_index])
```

Comment:
I think the style change might be caused by using different setting of pre-commit hooks. Can you check if you are using the correct one (the configuration in the root of the repo)?
There are multiple people working on this file and i want to prevent conflicts caused by style fixes.

Reply:
I am using black with line length 120, accordingly to the pre-commit hooks.

In the change the removed lines had a smaller line length and for this reason they have been moved to one line.